content/docs/700-other/150-myths.mdx

```diff
@@ -0,0 +1,86 @@
+
+## Effect will make your code 500x slower!
+
+That's pure FUD!
+
+Effect does perform 500x slower if you are comparing:
+
+```ts
+const result = 1 + 1
+```
+
+to 
+
+```ts
+const result = Effect.runSync(Effect.zipWith(
+  Effect.succeed(1),
+  Effect.succeed(1),
+  (a, b) => a + b
+))
+```
+
+The reason is one operation is optimized by the JIT compiler to be a direct CPU instruction and the other isn't. 
+
+In reality you'd never use Effect in such cases, Effect is an app-level library to tame concurrency, error handling, and much more! 
+
+You'd use Effect to coordinate your thunks of code, and you can build your thunks of code in the best perfoming manner as you see fit while still controlling execution through Effect.
+
+## Effect has a huge performance overhead!
+
+Depends what you mean by performance, many times performance bottlenecks in JS are due to bad management of concurrency.
+
+Thanks to structured concurrency and observability it becomes much easier to spot and optimize those issues. 
+
+There are apps in frontend running at 120fps that use Effect intensively, so most likely effect won't be your perf problem. 
+
+In regards of memory, it doesn't use much more memory than a normal program would, there are a few more allocations compared to non Effect code but usually this is no longer the case when the non Effect code does the same thing as the Effect code.
+
+The advise would be start using it and monitor your code, optimise out of need not out of thought, optimizing too early is the root of all evils in software design.
+
+## The bundle size is HUGE!
+
+Effect's minimum cost is about 25k of gzipped code, that chunk contains the Effect Runtime and already includes almost all the functions that you'll need in a normal app-code scenario.
+
+From that point on Effect is tree-shaking friendly so you'll only include what you use.
+
+Also when using Effect your own code becomes shorter and terser, so the overall cost is amortized with usage, we have apps where adopting Effect in the majority of the codebase led to reduction of the final bundle.
+
+## Effect is impossible to learn, there are so many functions and modules!
+
+True, the full Effect ecosystem is quite large and some modules contain 1000s of functions, the reality is that you don't need to know them all to start being productive, you can safely start using Effect knowing just 10-20 functions and progressively discover the rest, just like you can start using TypeScript without knowing every single NPM package.
+
+A short list of commonly used functions to begin are:
+
+- Effect.succeed
+- Effect.fail
+- Effect.sync
+- Effect.tryPromise
+- Effect.gen
+- Effect.runPromise
+- Effect.catchTag
+- Effect.catchAll
+- Effect.acquireRelease
+- Effect.acquireUseRelease
+- Effect.provide
+- Effect.provideService
+
+A short list of commonly used modules:
+
+- Effect
+- Context
+- Layer
+- Option
+- Either
+- Array
+- Match
```

Comment:
I'd probably leave out `Match` here, wdyt?

Reply:
Given ts-pattern I think that would fall into mainstream interest

Reply:
Fair. Personally, I don't use it that much on a regular basis compared to the others listed. But that's subjective I guess.